Makefile

```diff
@@ -1,4 +1,5 @@
 
+BROWSERIFY := node_modules/.bin/browserify
```

Comment:
Yeah, I prefer local dependencies. Gives you everything you need in an `npm install` and prevents "what version of Browserify are you running, oh, update/downgrade it"-type issues.

I generally just throw all my binaries at the top of the file because I've has issues with the `PATH` solution in the past (can't remember what those were), I'm open to whatever though